components/ContactCard/ContactCard.tsx

```diff
@@ -34,54 +38,121 @@ function ContactCard({ node }: Props) {
                     />
                 )}
                 <div>
-                    <h4 className="text-lg text-neutral-200 font-semibold mb-1">{name}</h4>
+                    <h4 className="text-lg text-neutral-200 font-semibold">{name}</h4>
                     {subtitle && <h5 className="text-neutral-400">{subtitle}</h5>}
                 </div>
             </div>
-            <div className="mt-2 flex items-center flex-wrap">
-                {email && (
-                    <a href={`mailto:${email}`} className="default-link mr-6 flex items-center">
-                        <IconEmail className="mr-2" width={16} height={16} />
-                        {email}
-                    </a>
-                )}
-                {website && (
-                    <a href={`${website}`} className="default-link mr-6 flex items-center">
-                        <IconGlobe className="mr-2" width={16} height={16} />
-                        {website}
-                    </a>
-                )}
-                {mobile && (
-                    <a href={`tel:${mobile}`} className="default-link mr-6 flex items-center">
-                        <IconMobile className="mr-2" width={16} height={16} />
-                        {mobile}
-                    </a>
-                )}
-                {phone && (
-                    <a href={`tel:${phone}`} className="default-link mr-6 flex items-center">
-                        <IconPhone className="mr-2" width={16} height={16} />
-                        {phone}
-                    </a>
-                )}
-                {twitter && (
-                    <a
-                        href={`https://twitter.com/${twitter}`}
-                        className="default-link mr-6 flex items-center"
-                    >
-                        <IconTwitter className="mr-2" width={16} height={16} />
-                        {`@${twitter}`}
-                    </a>
-                )}
-                {facebook && (
-                    <a
-                        href={`https://facebook.com/${twitter}`}
-                        className="default-link mr-6 flex items-center"
-                    >
-                        <IconFacebook className="mr-2" width={16} height={16} />
-                        {`${facebook}`}
-                    </a>
-                )}
-            </div>
+            {isCard && (
```

Comment:
I think it would be cleaner to use two separate components for Card and Signature layout (maybe just extracting this content logic to not repeat the avatar and name part).

Reply:
Good point, I've refactored the whole component into `Contact` which conditionally renders either of the two variants: 936f8c9fdd9977a048e22116247b4866fefabcb8